src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -161,8 +155,7 @@ export class CalciteAccordionItem {
 
   @Listen("calciteAccordionChange", { target: "body" })
   updateActiveItemOnChange(event: CustomEvent): void {
```

Comment:
Nitpick: I think you could do this to avoid casting below: `updateActiveItemOnChange(event: CustomEvent<HTMLCalciteAccordionItemElement>): void {`